atomdb/datasets/numeric/run.py

```diff
@@ -31,13 +31,17 @@
 
 import atomdb
 
-from atomdb.utils import MULTIPLICITIES
+from atomdb.utils import MULTIPLICITIES, DEFAULT_DATAPATH
 
 from atomdb.periodic import Element
 
 
-def load_numerical_hf_data():
+def load_numerical_hf_data(data_path=None):
```

Comment:
why not  `def load_numerical_hf_data(data_path=DEFAULT_DATAPATH):` ?

Reply:
Even more, I think this function should not have a default value. The only place where it is used inside `run` (line 183), a value is passed to this argument. I think the function that could/should have a default value should be the run function (line 155).

Reply:
Good point!
I think I'll go along the lines of your first suggestion making the change to simply
`def load_numerical_hf_data(data_path)`

I wouldn't make the change to the `run` function to keep consistency across all compilation scripts, and since the default path gets defined as kwd argument in the `load` function from the species module:
https://github.com/theochem/AtomDB/blob/511a7026725a833d068aca55d9c89f1dd26f735d/atomdb/species.py#L782-L790
The way I see it, the `run` functions will hardly ever get used directly by the user, so having the default value of datapath only in `load` seems OK to me.

Reply:
Looks OK to me. If @marco-2023 and @gabrielasd are happy with it's fine to merge IMHO.